inst/examples/germany_parametric_with_non_parametric_effect.R

```diff
@@ -93,7 +93,7 @@ expectation_module <- enw_expectation(
 # may not be the case
 reference_module <- enw_reference(
   parametric = ~ 1,
-  non_parametric = ~ 1 + delay0mod,
+  non_parametric = ~ 0 + delay0mod,
```

Comment:
This was previously mistakenly updated and I included the fix here. I don't really think its worth putting in its own PR...